docs/getting-started/using-gnoland.md

```diff
@@ -0,0 +1,543 @@
+via the CLI. Below are the available subcommands and their uses.
+
+### Initializing Secrets
+
+If you wish to configure and manage your node manually, you can choose to initialize
+the validator private key, the node p2p key, and the validator's last sign state
+using the `secrets init` subcommand.
+
+```bash
+gnoland secrets init [flags] [<key>]
+```
+
+Running the `init` subcommand without any flags will generate a default data
+directory for node secrets. Let's see it in action:
+
+```bash
+> gnoland secrets init
+Validator private key saved at gnoland-data/secrets/priv_validator_key.json
+Validator last sign state saved at gnoland-data/secrets/priv_validator_state.json
+Node key saved at gnoland-data/secrets/node_key.json
+```
+
+#### Flags
+- `-data-dir gnoland-data/secrets` - the directory where node secrets will be saved to
+- `-force` - if some secrets already exist, they will be overwritten with new ones
+
+### Getting Secrets
+
+To access the public values of your gno.land node secrets through the CLI, you
+can use the `secrets get` subcommand:
+
+```bash
+gnoland secrets get [flags] [<key>]
+```
+
+Getting all values can be done with the following command:
+
+```bash
+❯ gnoland secrets get
+
+[Node P2P Info]
+
+Node ID:  g16942vugc4g98j6gqfm94tnd4qwwgk8ee4nuntd
+[Validator Key Info]
+
+Address:     g1ukvflgxqehyrzwm0ulh7a9sn6hm7505dc0mwrj
+Public Key:  gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zquv55ka63cpwqx56j22fesrevem54dqdlfy2s008l4ev2cvhnnyk9tdf0t
+[Last Validator Sign State Info]
+
+Height:  0
+Round:   0
+Step:    0
+```
+
+#### Flags
+- `-data-dir gnoland-data/secrets` - the directory from which to read the secrets
+
+### Verifying Secrets
+
+You can verify the existence and integrity of all node secrets using the 
+`secrets verify` subcommand. The command can be run for any specific secret,
+or for all of them at the same time.
+
+```bash
+gnoland secrets verify [flags] [<key>]
+```
+
+In case a specific node secret is missing or corrupt, the command will let you know.
+For example, if the `node_key.json` file was missing from the secrets directory,
+checking for it would result in the following output:
+
+```bash
+❯ gnoland secrets verify NodeKey 
+unable to read node p2p key, unable to read data, open gnoland-data/secrets/node_key.json: no such file or directory
+```
+
+#### Flags
+- `-data-dir gnoland-data/secrets` - the directory from which to read the secrets
+
+## Node Configuration
+
+The `gnoland config` command suite helps manage the gno.land node configuration.
+
+### Initializing Configuration
+
+To initialize the gno.land node configuration, you can use the following command:
+
+```bash
+gnoland config init [flags]
+```
+
+This command will initialize a configuration directory and file in the default
+path. The node is highly configurable. Some of the main configuration options
+which are available are listed below:
+- Database which is used for storing node data
```

Comment:
I'm not sure people care about all these in a getting started section; a link to a reference page with this information might be better.

Reply:
Coming back to this after reading the rest of this section -- it would be great to have this link to a doc that shows all the available categories and fields.